subsys/caf/modules/power_manager.c

```diff
@@ -77,6 +81,84 @@ static void power_down_counter_abort(void)
 	LOG_DBG("Power down timer aborted");
 }
 
+static int settings_set(const char *key, size_t len_rd, settings_read_cb read_cb, void *cb_arg)
```

Comment:
please place these storage related functions as first local functions in this file